Demos/02-add-aad-auth/graph-tutorial/PrivateSettings.config.example

```diff
@@ -1,11 +1,6 @@
-﻿<?xml version="1.0"?>
-
-<!-- Copyright (c) Microsoft Corporation. All rights reserved.
-     Licensed under the MIT License. -->
-
-<appSettings>
-    <add key="ida:AppID" value="YOUR_APP_ID" />
-    <add key="ida:AppSecret" value="YOUR_APP_PASSWORD" />
-    <add key="ida:RedirectUri" value="https://localhost:44375/" />
+﻿<appSettings>
+    <add key="ida:AppID" value="YOUR APP ID" />
+    <add key="ida:AppSecret" value="YOUR APP PASSWORD" />
+    <add key="ida:RedirectUri" value="https://localhost:PORT/" />
```

Comment:
Not really needed here as if the user downloads the project they get the port already set in the project.

Reply:
I'm not clear... all these settings are needed by the app otherwise it doesn't know what AzureAD app to use. If we delete this, how does the app know this?

Reply:
Sorry, I meant changing the actual port number to `PORT`. Because they only time someone uses this file is if they download from GitHub, and the port is fixed there.